lms/djangoapps/teams/tests/test_views.py

```diff
@@ -129,7 +129,7 @@ def test_query_counts(self):
         team.add_user(self.user)
 
         # Check the query count on the dashboard again
-        with self.assertNumQueries(19):
+        with self.assertNumQueries(20):
```

Comment:
@nasthagiri 
about assertNumQueries(20) in teams test; 
we initially replaced loop with `UserPreference.get_value(user, 'account_privacy')` for the serializer `UserReadOnlySerializer` that increased the query count because it is being referenced in teams app too. Do we prefetch `preference` in all applications that are making use of (or may be make use of infuture) this Serializer?